docs/kubeadm.md

```diff
@@ -0,0 +1,249 @@
+    <util/kubeconfig> created "/etc/kubernetes/admin.conf"
+    <master/apiclient> created API client configuration
+    <master/apiclient> created API client, waiting for the control plane to become ready
+    <master/apiclient> all control plane components are healthy after 61.346626 seconds
+    <master/apiclient> waiting for at least one node to register and become ready
+    <master/apiclient> first node is ready after 4.506807 seconds
+    <master/discovery> created essential addon: kube-discovery
+    <master/addons> created essential addon: kube-proxy
+    <master/addons> created essential addon: kube-dns
+
+    Kubernetes master initialised successfully!
+
+    You can connect any number of nodes by running:
+
+    kubeadm join --token <token> <master-ip>
+
+Make a record of the `kubeadm join` command that `kubeadm init` outputs.
+You will need this in a moment.
+The key included here is secret, keep it safe &mdash; anyone with this key will be able to add authenticated nodes to your cluster.
+
+The key is used for mutual authentication between the master and the joining nodes.
+
+### (3/4) Joining your nodes
+
+The nodes are where your containers (your workload) will run.
+If you want to add any new machines as nodes to your cluster, for each machine: SSH to that machine, become root (e.g. `sudo su -`) and run the command that was output by `kubeadm init`.
+For example:
+
+    # kubeadm join --token <token> <master-ip>
+    <util/tokens> validating provided token
+    <node/discovery> created cluster info discovery client, requesting info from "http://138.68.156.129:9898/cluster-info/v1/?token-id=0f8588"
+    <node/discovery> cluster info object received, verifying signature using given token
+    <node/discovery> cluster info signature and contents are valid, will use API endpoints [https://138.68.156.129:443]
+    <node/csr> created API client to obtain unique certificate for this node, generating keys and certificate signing request
+    <node/csr> received signed certificate from the API server, generating kubelet configuration
+    <util/kubeconfig> created "/etc/kubernetes/kubelet.conf"
+
+    Node join complete:
+    * Certificate signing request sent to master and response
+      received.
+    * Kubelet informed of new secure connection details.
+
+    Run 'kubectl get nodes' on the master to see this machine join.
+
+A few seconds later, you should notice that running `kubectl get nodes` on the master shows a cluster with as many machines as you created.
+
+**YOUR CLUSTER IS NOT READY YET!**
+
+Before you can deploy applications to it, you need to install a pod network.
+
+### (4/4) Installing a pod network
+
+You must install a pod network add-on so that your pods can communicate with eachother when they are on different hosts.
+
+Several projects provide Kubernetes pod networks.
+A simple one with no infrastructure or database dependencies is Weave Net, which you can install by running, on the master:
```

Comment:
Based on user testing of this doc so far I think it's important to have one recommended option that is marked "easy to install", and I'd selfishly like that to be Weave Net. However it's also important to emphasize that the user has choice, so I will add an explicit call-out to Calico and Canal in the text. @caseydavenport if you can give me some words to use to describe them and the distinction between them and somewhere to link to, I'll add that here. Thanks!